This is a testharness.js-based test.
FAIL Backspace in "<dl contenteditable><dt>{}</dt></dl>" promise_test: Unhandled rejection with value: object "Error: element send_keys intercepted error"
FAIL Backspace in "<dl contenteditable><dd>{}</dd></dl>" promise_test: Unhandled rejection with value: object "Error: element send_keys intercepted error"
FAIL Backspace in "<dl contenteditable><dd><ul><li>{}</li></ul></dd></dl>" assert_in_array: value "<dd><div><br></div></dd>" not in array ["<dd></dd>", "<dd><br></dd>"]
FAIL Backspace in "<dl contenteditable><dd><ol><li>{}</li></ol></dd></dl>" assert_in_array: value "<dd><div><br></div></dd>" not in array ["<dd></dd>", "<dd><br></dd>"]
FAIL Backspace in "<dl contenteditable><ul><li>{}</li></ul></dl>" assert_in_array: value "<div><br></div>" not in array ["<dd></dd>", "<dd><br></dd>"]
FAIL Backspace in "<dl contenteditable><ol><li>{}</li></ol></dl>" assert_in_array: value "<div><br></div>" not in array ["<dd></dd>", "<dd><br></dd>"]
FAIL Backspace in "<dl contenteditable><dl><dt>{}</dt></dl></dl>" promise_test: Unhandled rejection with value: object "Error: element send_keys intercepted error"
FAIL Backspace in "<dl contenteditable><dl><dd>{}</dd></dl></dl>" promise_test: Unhandled rejection with value: object "Error: element send_keys intercepted error"
Harness: the test ran to completion.

